### DISCUSSION TITLE
When will you add pagination support ?

### DISCUSSION THREAD

**Discussion by iamleson98** [JordanKnott/taskcafe#94] [Q&A]
Hey @JordanKnott , I have subscribed to this repository and studied it for month now. I notice you have not add pagination support to it. I am trying to borrow relay style from python/graphene but seem like i cannot figure it out soon(I still try). 
You know there are 2 popular pagination style (cursor and pages), people say cursor style result in better result and relay adopts it.
What do you think and how would you add pagination to this project ?

Thank you!

**Comment by JordanKnott** [comment]
Hey thanks for using the discussion feature!

I haven't looked much into pagination yet (like at all) as I haven't really hit the point where I need it (at least in my usage). Once I start hitting that limit where I can longer just fetch everything in one call, that's when I'll look into pagination (or if it becomes a problem for many others).

So while I can't say much on how I'd specifically add it to the project, based on past experience, I'll likely go with whatever solution integrates easier with React 🤷‍♂️
